cmd/data-scanner.go

```diff
@@ -1343,11 +1342,8 @@ func (d *dynamicSleeper) Update(factor float64, maxWait time.Duration) error {
 }
 
 func auditLogLifecycle(ctx context.Context, bucket, object string) {
-	entry := audit.NewEntry(globalDeploymentID)
-	entry.Trigger = "internal-scanner"
-	entry.API.Name = "DeleteObject"
-	entry.API.Bucket = bucket
-	entry.API.Object = object
-	ctx = logger.SetAuditEntry(ctx, &entry)
-	logger.AuditLog(ctx, nil, nil, nil)
+	auditLogInternal(ctx, bucket, object, AuditLogOptions{
+		Trigger: "internal-scanner",
```

Comment:
```suggestion
		Trigger: "ilm",
```

Reply:
This can be `ilm:expiry` and transition can have `ilm:transition` instead of `internal-scanner`